nupic/research/temporal_memory.py

```diff
@@ -233,7 +258,16 @@ def activateCorrectlyPredictiveCells(self,
         winnerCells.add(cell)
         predictedColumns.add(column)
 
-    return activeCells, winnerCells, predictedColumns
+    if self.permanenceOrphanDecrement>0:
+      # If any cell was previously matching and we don't get bottom up activity
+      # in that column, mark it as an orphan cell.
```

Comment:
I don't think this comment is necessary, since the code below is self-documenting.